Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -418,6 +447,18 @@ public class Fuzzer {
         return corpus.supportsFastStateSynchronization
     }
 
+    private func shouldExecutionBeIgnoredForDifferentialTesting(execution: Execution) -> Bool {
+        return config.differentialPoison.contains { poison in
```

Comment:
I updated the code s.t. `shouldExecutionBeIgnoredForDifferentialTesting` is always called because even if a particular program doesn't include a differential test. This is necessary because the engine might be instantiated with command line flags for differential testing and crash because it detected a non-recoverable non-determinism.